webapp/app/appModules.js

```diff
@@ -121,7 +121,7 @@ export const dataModules = {
 export const userModules = {
   users: {
     key: 'userList',
-    path: `${appModules.users.path}/list`,
+    path: `${appModules.users.path}/userList`,
```

Comment:
any reason to change the path?

Reply:
it is the only module in which the key is not the same as the submodule. is there any reason for that? 

```
export const userModules = {
  users: {
    key: 'userList',
    path: `${appModules.users.path}/list`,
  },
  user: {
    key: 'user',
    path: `${appModules.users.path}/user`,
  },
  userInvite: {
    key: 'userInvite',
    path: `${appModules.users.path}/userInvite`,
  },
}
```